docs/en/04_Changelogs/3.7.0.md

```diff
@@ -0,0 +1,29 @@
+# 3.7.0 (unreleased)
+
+## SilverStripe 3.7 and PHP 7.2 and Object subclasses
+
+SilverStripe 3.7 now supports PHP 7.2, which is exciting, but PHP 7.2 introduces an `object` keyword.
+On a custom project, you can simply replace any uses of `Object` with `SS_Object`.
+
+```diff
+-class MyClass extends Object
++class MyClass extends SS_Object
+{
+	public function myFunction()
+	{
+-		$foo = Object::has_extension('MyExtension');
++		$foo = SS_Object::has_extension('MyExtension');
+	}
+}
+```
+
+If you are publishing a 3.x-compatible module that directly references to the `Object` class,
+it's a bit more complicated. You can apply one of two fixes:
+
+ * Mark your module as not working with PHP 7.2, by putting `"php": ">=5.3, <7.2"` in your `composer.json`.
+ * Create a new version of your module that requires `"silverstripe/framework": "^3.7"`.
+   In this version, replace all references to `Object` with `SS_Object`.
+   This version won’t work on SilverStripe 3.6,
+   but it will work on SilverStripe 3.7+ on any version of PHP.
+   Note that in SilverStripe 4.x, the `Object` class was deleted so there isn’t an `SS_Object` class there either
+   (see [https://docs.silverstripe.org/en/4/changelogs/4.0.0/](changelog))
```

Comment:
Do you think there's a third option which is to avoid using the `Object` parent class altogether? This could be doable in some situations but not always, i.e. a utility class can be loaded via `Injector::inst()->get(MyClass::class)` rather than `MyClass::singleton()`

Reply:
Can you list a few before/after cases which make `Object` obsolete?

Reply:
Perhaps it's not worth mentioning... you'd lose all the extensibility by not extending Object/SS_Object. Ignore me =)